runtime-watcher/tests/e2e/utils/builder.go

```diff
@@ -0,0 +1,104 @@
+package utils
```

Comment:
Can we reuse `github.com/kyma-project/lifecycle-manager/pkg/testutils/builder`?

Reply:
I tried to use it, it pulls in too many sub dependencies. And we are just using a Kyma here.
Almost considering just deleting the builder here, and create this one Kyma CR directly.